repo_id,repo_full_name,file,file_sha1,match
362588615,lanzt/CVE-2020-14321,README.md,50513f5ebff0eb267a17dd0f0e1c4ac2d10fa4a1,CVE-2020-14321
362588615,lanzt/CVE-2020-14321,CVE-2020-14321_RCE.py,845c1410f080dbcf82b78fb047ab2bc46d5a4668,CVE-2020-14321
362588615,lanzt/CVE-2020-14321,_GITHUB_REPO_METADATA_,,CVE-2020-14321
